### ISSUE TITLE
Error in H5Fopen("merge_graphs_exon_skip_C3.counts.hdf5") :    HDF5. File accessibility. Unable to open file.

### ISSUE THREAD

**Issue by ls233** [R-Najjar/RiboSplitter#2]
with rhdf5 v.2.46.1 under R v4.3, I keep getting the following error:

Error in H5Fopen("merge_graphs_exon_skip_C3.counts.hdf5") :    HDF5. File accessibility. Unable to open file.

Same error with: H5Fopen("merge_graphs_exon_skip_C3.counts.hdf5")

a bit of debugging:

I can list the file, so it is accessible:
> h5ls("merge_graphs_exon_skip_C3.counts.hdf5")
   group           name       otype  dclass           dim
0      /       conf_idx H5I_DATASET INTEGER         23359
1      /      confirmed H5I_DATASET INTEGER         52586
2      /   event_counts H5I_DATASET   FLOAT 52586 x 7 x 4

and a bit more verbose error:
> h5errorHandling(type = "verbose")
> hf1= H5Fopen("merge_graphs_exon_skip_C3.counts.hdf5")
Error in H5Fopen("merge_graphs_exon_skip_C3.counts.hdf5") : libhdf5
    error #000: H5F.c in H5Fopen(): line 429
        class: HDF5
        major: File accessibility
        minor: Unable to open file
    error #001: H5Fint.c in H5F_open(): line 1644
        class: HDF5
        major: File accessibility
        minor: Unable to open file
    error #002: H5FD.c in H5FD_open(): line 741
        class: HDF5
        major: Virtual File Layer
        minor: Unable to initialize object
    error #003: H5FDsec2.c in H5FD__sec2_open(): line 360
        class: HDF5
        major: File accessibility
        minor: Unable to open file

and I can access a data slot directly with:
iso1= h5read("merge_graphs_exon_skip_C3.counts.hdf5","iso1")

**Comment by R-Najjar** [comment]
This seems like an HDF5 issue. Are you sure you're using the right directory? You can try typing the full path and see if that works

**Comment by R-Najjar** [comment]
I will close this, let me know if you have more questions
